api_tests/base/test_sloan_study.py

```diff
@@ -57,6 +58,7 @@ def preprint(self, user):
     def providers(self, user):
         PreprintProviderFactory(_id='foorxiv').save()
         PreprintProviderFactory(_id='osf').save()
+        PreprintProviderFactory(_id='burdixiv', domain='https://burdixiv.burds/').save()
```

Comment:
This line causes [these lines](https://github.com/CenterForOpenScience/osf.io/pull/9338/files#diff-35f5a3f49524bbaf7aee4d9e75149182L164-L166) to be removed.